libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -86,6 +106,22 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     }
   }
 
+  @HostListener('window:keyboardWillShow')
+  _onKeyboardWillShow() {
+    this.keyboardVisible = true;
+  }
+
+  @HostListener('window:keyboardWillHide')
```

Comment:
I believe that the `keyboardWillHide`-event is dispatched by cordova (capacitor), but won't we experience "the same wierdness" when running on a mobile browser?

Reply:
@askarby Yep, this is a "temporary" fix for the native app - we are nowhere near supporting PWAs on native devices yet...